ra_aid/file_listing.py

```
@@ -1,8 +1,11 @@
 """Module for efficient file listing using git."""
 
 import subprocess
+import os
 from pathlib import Path
 from typing import List, Optional, Tuple
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)